### DISCUSSION TITLE
Nicktoons UK

### DISCUSSION THREAD

**Discussion by nickelodeonbiy** [iptv-org/community#754] [Q&A]
### Channel Name

Nicktoons

### Channel ID

Nicktoons.uk

### Country

England

### Language

English 

### Website

http://tv247.us/watch/nicktoons/

**Comment by nickelodeonbiy** [comment]
Let me now if you find a link

**Comment by BellezaEmporium** [comment]
http://live.tv247.club/live/nicktoons.m3u8 referer locked to http://tv247.us/

**Comment by freearhey** [comment]
@MapGuy11 I already explained that "the label indicates the file to which the channel will need to be added, not the country of broadcast." (https://github.com/iptv-org/iptv/issues/5011#issuecomment-949944638)

**Comment by MapGuy11** [comment]
> @MapGuy11 I already explained that "the label indicates the file to which the channel will need to be added, not the country of broadcast." ([#5011 (comment)](https://github.com/iptv-org/iptv/issues/5011#issuecomment-949944638))

@freearhey 
But it's primarily owned by Sky UK for the UK division whereas the US division is owned by ViacomCBS. Also owned by a different division of ViacomCBS for the UK division.
UK:

ViacomCBS Networks UK & Australia (60%)
Sky Group (40%)

US:

ViacomCBS Domestic Media Networks

**Comment by freearhey** [comment]
@MapGuy11 The whole point of splitting channels into files is just to avoid creating a bunch of duplicates. So, wherever possible, it is better to collect the channels in one file. 

If you're not sure which file to put the channel in, just use the description on lyngsat or wikipedia: https://www.lyngsat.com/tvchannels/us/NickToons-UK.html

**Comment by freearhey** [comment]
http://live.tv247.club/live/nicktoons.m3u8 (offline)
<img width="299" alt="image" src="https://user-images.githubusercontent.com/7253922/232223194-5c18ef4e-fc5d-4ef1-8d32-041b58b55a1f.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19768
